packages/generator-react-server/test/app.js

```diff
@@ -68,8 +68,13 @@ function exists(filename, dir) {
 function runsSuccessfully(command, dir) {
 	return new Promise((resolve) => {
 		cp.exec(command, {
-			cwd: dir
-		}, (error) => {
+			cwd: dir,
+		}, (error, stdout, stderr) => {
+			if (error) {
+				console.error(error);
+				console.error(stdout);
+				console.error(stderr);
+			}
```

Comment:
This is just to see errors that come out of the `npm test` inside the generated package.